hw2/ViewModel/MatchViewModel.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  MatchViewModel.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class MatchViewModel {
```

Comment:
struct MatchViewModel

class ubiti koristimo samo di nam treba nasljeđivanje jer strukture to ne mogu ostvariiti. Npr. UIViewController, UIView...

Možeš pogledati glavne razlike struct vs. classes
https://medium.com/@muhammad.cse11/class-vs-struct-in-swift-dcc7ad6f5a99#:~:text=Use%20classes%20when%20you%20need,lightweight%20data%20structures%2C%20or%20immutability.

Reply:
Inače, bravo za viewModel, dobro definiran!